java/arcs/core/util/Scheduler.kt

```diff
@@ -146,7 +146,9 @@ class Scheduler(
     private suspend fun executeAgenda(agenda: Agenda) {
         val timeoutHandler = { throwable: Throwable ->
             if (throwable is TimeoutCancellationException) {
-                log.error(throwable) { "Scheduled tasks timed out." }
+                // TODO(b/160251910): Make logging detail more cleanly conditional.
+                log.debug(throwable) { "Scheduled tasks timed out." }
```

Comment:
Since Scheduler's are tied to to Arcs, it might be worthwhile to log the context[CoroutineName] when debugging so you'd see something like "Scheduled tasks timed out for {arcId}"

Reply:
Ah, sorry I missed this. Yes, we can add whatever we want for debug/verbose. I'll look to do this one tomorrow.